openstack/run_packstack_docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+docker run -d --rm --name packstack  \
+-v /lib/modules/:/lib/modules/ \
+--privileged --cap-add ALL --security-opt seccomp=unconfined   --cgroup-parent=docker.slice \
+-p 0.0.0.0:5000:5000 \
+-p 0.0.0.0:8774:8774 \
+-p 0.0.0.0:8775:8775 \
+-p 0.0.0.0:8778:8778 \
+-p 0.0.0.0:9292:9292 \
+-p 0.0.0.0:9696:9696 \
+quay.io/eslutsky/packstack:latest
```

Comment:
ah and please move it to kubev2v and then I'll configure https://quay.io/kubev2v/packstack to point to it

Reply:
so i'll also move  the dockerfile and relevant build script to the forkliftci repo?

Reply:
I think it would be better to have it in a separate repo, like https://github.com/kubev2v/fakeovirt, and we'll configure quay.io to build the image every time this repo changes

Reply:
created repo here https://github.com/kubev2v/packstack-img

Reply:
updated deployment to quay.io/kubev2v/packstack:latest